src/_data/catalog/widgets.json

```diff
@@ -235,6 +235,17 @@
     "link": "https://docs.flutter.io/flutter/material/IconButton-class.html",
     "image": "<img alt='' src='https://storage.googleapis.com/material-design/publish/material_v_11/assets/0B_udO5B8pzrzdXVuTlBoOTBjcU0/components_buttons_other1.png'>"
   },
+  {
+    "name": "DropdownButton",
+    "sample": "DropdownButton",
```

Comment:
I'm not seeing this field being used anywhere. Consider dropping it.

Reply:
The `sample` field is in a bunch of the entries (Scaffold, AppBar), including PopupMenuButton, which I copied. Should I delete it for just this one and/or file an issue to look into the `sample` fields?

Reply:
Filing an issue is probably the best option, thanks.

Reply:
I'll drop it for this one, since I don't know that a sample exists AND file that issue.